### ISSUE TITLE
1.6 error in comment handling inside of strings

### ISSUE THREAD

**Issue by dieterm57** [arduino/Arduino#2641]
Following code does not compile:
```
  Serial.println("/\\");
  Serial.println("//");
```

but insert a comment in between:
```
  Serial.println("/\\");
  //\
  Serial.println("//");
```
... and it does
wtf?

**Comment by matthijskooijman** [comment]
Could you provide the compiler error messages you are getting?

**Comment by dieterm57** [comment]
![editor](http://i.imgur.com/cotWDc2.png)

```
sketch_feb13b.ino:10:3: error: missing terminating " character
sketch_feb13b.ino: In function 'void loop()':
sketch_feb13b.ino:11:1: error: expected primary-expression before '}' token
Error compiling.
```

**Comment by ffissore** [comment]
This is yet another bug related to the preprocessor. Please try one of the builds from #2636

**Comment by ffissore** [comment]
Fixed by #3779
